bin/generate.js

```diff
@@ -47,6 +48,19 @@ mkdirp(outputPath, err => {
     fs.copyFileSync(logoPath, path.join(outputPath, 'logo.png'));
   }
 
+  try {
```

Comment:
I'm pretty sure that not specifying a `--data-root` will result in the script replacing DATAROOT with `"undefined"` (string). This would break behavior, so before this try-catch, I'd check if `dataRoot` was provided.

Reply:
Nevermind this :)